resources/sdk/pureclouddotnet/templates/README.mustache

```diff
@@ -2,8 +2,8 @@
 title: Platform API Client SDK - .NET
 ---
 [![NuGet Badge](https://buildstats.info/nuget/PureCloudPlatform.Client.V2)](https://www.nuget.org/packages/PureCloudPlatform.Client.V2/)
-
```

Comment:
I'm 98% sure this line needs to stay there. Markdown concatenates inline content on consecutive lines. The blank line denotes a paragraph break, which will separate the image from the following text.